modules/virt-retrieving-vm-definition-from-vme-manifest.adoc

```diff
@@ -19,14 +19,37 @@ After you export a virtual machine (VM) or snapshot, you can access the `Virtual
 
 .Procedure
 
-. Get the `VirtualMachineExport` custom resource by running the following command:
+. To access the manifests, you must first copy the certificates from the source cluster to the target cluster.
+
+.. Log in to the source cluster.
+
+.. Save the certificates to the file `cacert.crt` by running the following command:
 +
 [source,terminal]
 ----
-$ oc get vmexport <export_name> -o yaml <1>
+$ oc get vmexport <export_name> -o jsonpath={.status.links.external.cert} > cacert.crt <1>
 ----
 <1> Replace `<export_name>` with the `metadata.name` value from the `VirtualMachineExport` object.
 
+.. Copy the `cacert.crt` file to the target cluster.
+
+. Decode the token in the source cluster and save it to the file `token_decode` by running the following command:
++
+[source,terminal]
+----
+$ oc get secret export-token-<export_name> -o jsonpath={.data.token} | base64 --decode > token_decode <1>
+----
+<1> Replace `<export_name>` with the `metadata.n
```

Comment:
<1> Replace `<export_name>` with the `metadata.name` value from the `VirtualMachineExport` object.
(maybe we can even remove it and '<1>' from the line above because we say it in line 32, but whatever you think is better)

Reply:
I have no idea how that got cut off! 😹 Great catch. I think I'll fix it and leave it in, just because it's a different command and folks might not be reading super closely